### ISSUE TITLE
design contribution

### ISSUE THREAD

**Issue by masudr4n4** [provswin/OKEx-Monitor#2]
hallo 
great to know your projects and wants to make a logo for this projects.its free for open source projects.if you have any requirements then let me know.

**Comment by provswin** [comment]
Yes, i'm trying to find somebody design a logo for this project.
if you are free, please design a logo for this project.
thanks , you are very kind.
you can refer to the website : https://www.okex.com/

**Comment by provswin** [comment]
@maaudrana 
Yes, i'm trying to find somebody design a logo for this project.
if you are free, please design a logo for this project.
thanks , you are very kind.
you can refer to the website : https://www.okex.com/

**Comment by masudr4n4** [comment]
if you need anything change please comment blow

![okex-monitor-01](https://user-images.githubusercontent.com/34313493/40282715-4c28f4b0-5c95-11e8-9850-0defbafeef26.png)
